#!/usr/bin/env python
PACKAGE = 'robotic_arm'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('ratio_3510', double_t, 0, 'description', -0.0, -1000.0, 1000.0)
gen.add('ratio_6020', double_t, 0, 'description', -0.0, -1000.0, 1000.0)

exit(gen.generate(PACKAGE, 'robotic_arm', 'MotorHelper'))